Code/GraphMol/MolStandardize/Wrap/rdMolStandardize.cpp

```diff
@@ -124,6 +191,28 @@ void mtinPlaceHelper(python::object pymols, int numThreads,
     func(mols, numThreads, *ps);
   }
 }
+template <typename FUNCTYPE>
+void mtinPlaceHelper2(python::object pymols, int numThreads,
+                      python::object params, bool skip_standardize,
+                      FUNCTYPE func) {
+  const RDKit::MolStandardize::CleanupParameters *ps =
```

Comment:
```suggestion
  const auto ps =
```

Reply:
this one actually isn't so easy since we plan to reassign `ps` later. It needs to be `const auto *ps`